Makefile

```diff
@@ -38,7 +38,7 @@ push-latest-docker-image: ## push the docker image with the "latest" tag to the
 
 .PHONY: git-tag
 git-tag: ## tag the current commit with the next version and push
-	git tag $(VERSION) &&\
+	git tag $(shell ./bin/git-version) &&\
```

Comment:
`$(shell ./bin/git-version)`

This is repeated 3 times already. Maybe add a variable up there for this

Reply:
the problem of having a variable is that it will be computed even when the binary is not there, If you have a better solution for having a `lazy` value in a Makefile

Reply:
done